CMakeLists.txt

```diff
@@ -1,103 +1,146 @@
 cmake_minimum_required(VERSION 3.1.0)
 
 project(pomerol CXX)
-message("\n${PROJECT_NAME} configuration\n")
-string(TOUPPER ${PROJECT_NAME} PROJECT_CAPNAME)
+set(POMEROL_VERSION 2.0)
+set(POMEROL_DESCRIPTION "An exact diagonalization library aimed at \
+solving condensed matter models of interacting fermions")
+set(POMEROL_URL "http://pomerol.sourceforge.net/")
```

Comment:
Do we want to change this URL btw?

Reply:
Great idea!

Let me keep this URL as it is for now. In a later PR I will be revising the docs and will move them to GitHub Pages (other suggestions?) with automated deployment via GitHub Actions.